birdhouse/optional-components/test-cowbird-jupyter-access/test_cowbird_jupyter.py

```diff
@@ -0,0 +1,188 @@
+"""
+Script used to setup a test user along with different test files, for the test notebooks
+"test_cowbird_jupyter.ipynb" and "user_test_cowbird_jupyter.ipynb" found in the `PAVICS-e2e-workflow-test` repo.
+"""
+
+import json
+import os
+import requests
```

Comment:
This script basically only need to make http requests, I think any `pavics/workflow-tests` is fine, no need for the specific version `210216`.

And is this script a run once script or it has to continuously run and listen to connections?

If not, can we make it called only once by https://github.com/bird-house/birdhouse-deploy/blob/master/birdhouse/scripts/bootstrap-instance-for-testsuite instead?

Reply:
Run once and forget. 
However, the bootstrap script is not called by our Jenkins CI tests.
Only the PAVICS-E2E `testall` for running the notebooks are called on the instance swanned with this component added.

Reply:
> However, the bootstrap script is not called by our Jenkins CI tests.

I am very surprised the boostrap script is called call by your Jenkins CI tests.  That boostrap entrypoint was specifically created for CI pipeline to call, so that new boostrap steps can be added transparently, without having to change the CI steps.

Reply:
The bootstrap entrypoint is called.

http://daccs-jenkins.crim.ca/job/DACCS-iac-birdhouse/2255/console
[RESULT] bootstrap-instance-for-testsuite script finished. Exiting.